src/app/pages/products-list/card/product-card.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+
+import {MatButtonModule} from '@angular/material/button';
+import {MatCardModule} from '@angular/material/card';
+
+import {Product} from '../../../shared/products/product.interface';
+import {productsMock} from '../../../shared/products/products.mock';
+
+@Component({
+    selector: 'app-product-card',
+    standalone: true,
+    imports: [MatCardModule, MatButtonModule],
+    templateUrl: './product-card.component.html',
+    styleUrl: './product-card.component.css',
+})
+export class ProductCardComponent {
+    product: Product;
+
+    constructor() {
+        this.product = productsMock[0];
+    }
```

Comment:
Почему не сделал так?
```suggestion
    readonly product = productsMock[0];
```

И почему не стал помечать `product` как `readonly`, если он не изменяем?
```suggestion
    readonly product: Product;

    constructor() {
        this.product = productsMock[0];
    }
```

Reply:
> Не readonly, наверно потому что не привыкла в TS его использовать (забываю, что такое есть :) )

Рекомендую взять за хорошую практику)

> Почему через конструктор не знаю :) Теоретически наверно раньше информация в product появится, чем если в свойство присваивать, но тут смысла не было)

В действительности разницы не будет код в конструкторе и свойству эквивалентен, просто записывается по разному. Вот тут можно посмотреть как компилируется TS код и что получается по итогу.

МР закрою, но если появятся вопросы по комментариям выше - закидывай в телеграмм, с радостью отвечу)